R/run-model.R

```diff
@@ -113,7 +113,8 @@ hintr_run_model <- function(data, options, output_path = tempfile(),
   fit <- fit_tmb(tmb_inputs,
                  outer_verbose = ifelse(is.null(options$outer_verbose), FALSE, options$outer_verbose),
                  inner_verbose = ifelse(is.null(options$inner_verbose), FALSE, options$inner_verbose),
-                 max_iter = ifelse(is.null(options$max_iterations), 250, options$max_iterations))
+                 max_iter = ifelse(is.null(options$max_iterations), 250, options$max_iterations),
```

Comment:
while you're changing code around here:

* what is 250, and should that get a variable name
* prefer `if (is.null(options$max_iterations) 250 else options$max_iterations` or `options$max_iterations %||% 250` to `ifelse`

Reply:
Tidied that, 250 is just default max iterations. Same as the default on `fit_tmb` function itself.